CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # master
+1. New Reactive extension for `UIButton`: `titleColor(for state:)`.
```

Comment:
Can you put this below  _*Please put new entries at the top_?

(Also note that you'll need to merge in `master` to pick up the changes to this file.)